modules/testcontainers/build.gradle

```diff
@@ -15,19 +15,28 @@ sourceSets {
 }
 
 dependencies {
-  compile 'org.testcontainers:junit-jupiter:1.19.1'
-  compileOnly project(':microshed-testing-core')
-  testCompile 'org.eclipse.microprofile.rest.client:microprofile-rest-client-api:1.4.1'
-  testCompile 'org.slf4j:slf4j-log4j12:1.7.36'
-  testCompile 'org.testcontainers:mockserver:1.19.1'
-  testCompile project(':microshed-testing-core')
-  testImplementation 'org.junit.jupiter:junit-jupiter:5.10.1'
+    implementation 'org.testcontainers:junit-jupiter:1.19.1'
+    implementation 'org.junit.jupiter:junit-jupiter-api:5.10.1'
+    compileOnly project(':microshed-testing-core')
+    testImplementation 'org.eclipse.microprofile.rest.client:microprofile-rest-client-api:1.4.1'
+    testImplementation 'org.slf4j:slf4j-log4j12:1.7.36'
+    testImplementation 'org.testcontainers:mockserver:1.19.1'
+    testImplementation project(':microshed-testing-core')
```

Comment:
This should be removed otherwise it will be included as a transitive dependency, but the user needs to choose if they want `core` or `core-jakarta`
```suggestion
```

Reply:
Using testCompileOnly should make it compile and not pass it on as a transitive dep right?